licenses/management/commands/publish.py

```diff
@@ -96,7 +110,13 @@ def publish_all(self):
             self.publish_branch(b)
 
     def handle(self, *args, **options):
-        if options.get("list_branches"):
+        self.output_dir = options["output_dir"]
+        self.use_git = True
+        if options["nogit"]:
+            self.use_git = False
+            self.publish_branch(None)
+            return
```

Comment:
Is it fine in this use case to have a return statement in `handle`?

Reply:
It looks like a return is not necessary here, and different from the other branches of the if statement - I'll remove it to clean it up.